package.json

```diff
@@ -24,11 +24,18 @@
    },
    "homepage": "https://github.com/romppane/Bulletin_Board#readme",
    "dependencies": {
+      "@hapi/boom": "^7.4.2",
+      "@types/boom": "^7.2.1",
       "@types/express": "^4.17.0",
+      "@types/hapi__boom": "^7.4.0",
```

Comment:
Is types for boom installed twice? I am referring to `@types/hapi__boom` and `@types/boom`

Reply:
Looks like I forgot to uninstall @types/boom which I installed simultaniously with boom, which to my big surprice wasn't the real dependency needed? Leaving it uninstalled this time as I swap the dependencies to the dev mode though!